Hummingbird.docc/Tutorials/Todos/Resources/code/Postgres/todos-postgres-14.swift

```diff
@@ -41,3 +41,19 @@ public func buildApplication(_ arguments: some AppArguments) async throws -> som
     }
     return app
 }
+
+/// Build router
+func buildRouter(_ repository: some TodoRepository) -> Router<AppRequestContext> {
```

Comment:
Again the contents of buildRouter() isn't necessary here.

Instead it'd be better to make it clear we are only displaying `buildApplication()`. If you go into the Todos-4-Postgres.tutorial and find where we are displaying this code and set the title to be "buildApplication() - Sources/App/Application+build.swift" instead of "Sources/App/Application+build.swift".

Reply:
The signature of `buildRouter` needs to change in (or around) this step of the tutorial (`todos-postgres-14.swift`). The change appears in this step at the call site in `buildApplication()` but I couldn't find the updated declaration of `buildRouter` in any other steps of the tutorial. I got it from the examples directory. I apologize if it was there and I missed it.

Reply:
Ok fair point

Reply:
There is already quite a bit of code on that page. I wasn't keen to add more, but yes you do miss a small change to `buildRouter()`.

Reply:
Maybe we could add another step which includes `buildRouter()` code.

Reply:
Yeah that's a fair point. Let me try adding it as an additional step.